README.md

```diff
@@ -29,6 +31,22 @@ Upcoming:
 - see below for setting up search functionality
 
 
+### With Docker
+
+- Make sure that your system has Docker installed.
+
+If you wish to test the image for yourself, you can run
+
+```
+docker run -d --name archivy -p 5000:5000 -v /path/to/host/dir:/usr/src/app/data harshavardhanj/archivy:latest
+```
+and you can access the service at `http://localhost:5000/` on your browser. 
+
+Also, if you wish to build the image on your own, there are `Dockerfile`s present in the repository.
```

Comment:
Could you specify the two different Dockerfiles here and when to use them? 

I'll merge right after.

Thanks :+1: